Persistence/Database/SeedData.cs

```diff
@@ -45,40 +49,38 @@ public static void PopulateIfEmpty(IApplicationBuilder app)
 
         private static void populateSenders(DatabaseContext context)
         {
-            FileInfo excelFileInfo = new FileInfo($"{basePath}/{nameof(Sender)}s.xlsx");
-            using (var excelFile = new ExcelPackage(excelFileInfo))
+            FileInfo excelFileInfo = new($"{basePath}/{nameof(Sender)}s.xlsx");
+            using (ExcelPackage excelFile = new ExcelPackage(excelFileInfo))
             {
-                var sheet = excelFile.Workbook.Worksheets.First();
-                int idColumnPosition = EPPlusExtensions.GetColumnByName(sheet, "ID");
-                int usernameColumnPosition = EPPlusExtensions.GetColumnByName(sheet, "Username");
-                int passwordColumnPosition = EPPlusExtensions.GetColumnByName(sheet, "Password");
+                ExcelWorksheet? sheet = excelFile.Workbook.Worksheets.First();
+                int idColumnPosition = sheet.GetColumnByName("ID");
+                int usernameColumnPosition = sheet.GetColumnByName("Username");
+                int passwordColumnPosition = sheet.GetColumnByName("Password");
                 for (int i = 2; i <= sheet.Dimension.Rows; i++)
                 {
                     ulong id = Convert.ToUInt64(
                         sheet.Cells[i, idColumnPosition].Value.ToString());
                     string username = sheet.Cells[i, usernameColumnPosition].Value.ToString();
                     string password = sheet.Cells[i, passwordColumnPosition].Value.ToString();
                     // TODO: password hash.
-                    Sender sender = new Sender
-                    {
-                        Id = id, Username = username, HashedPassword = new byte[7]
-                    };
+                    Sender sender = new() { Id = id, Username = username, HashedPassword = new byte[7] };
```

Comment:
Well, I think that is not a good formating (if you consider writing single line if in multiple lines), but it's not a problem for me.